pulser-pasqal/pulser_pasqal/pasqal_cloud.py

```diff
@@ -110,8 +109,12 @@ def submit(
                     "The measurement basis can't be implicitly determined "
                     "for a sequence not addressing a single basis."
                 )
-            # The copy prevents changing the input sequence
-            sequence = copy.deepcopy(sequence)
+            # This is equivalent to performing a deepcopy
+            # All tensors are converted to arrays but that's ok, it would
+            # have happened anyway later on
+            sequence = Sequence.from_abstract_repr(
+                sequence.to_abstract_repr(skip_validation=True)
+            )
```

Comment:
That's the thing, for now we don't have a way to serialize the full computational graph so we won't support it. If you try to submit to a torch-based backend, two things can happen:
- Your tensors don't `require_grad`, in which case they don't even have to be tensors and this will work fine
- Some tensors `require_grad` and the serialization will fail (an error will be raised because you didn't detach). This is on purpose, it tells the user that differentiability is not supported on remote backends (at least for now)